porcelain/miner.go

```diff
@@ -24,7 +24,7 @@ import (
 type mcAPI interface {
 	ConfigGet(dottedPath string) (interface{}, error)
 	ConfigSet(dottedPath string, paramJSON string) error
-	GetAndMaybeSetDefaultSenderAddress() (address.Address, error)
+	WalletDefaultAddress() (address.Address, error)
```

Comment:
please sort this lexicographically more towards the bottom of this list. this placement made sense when it started with 'g', but now it starts with 'w'.